CHANGES.md

```diff
@@ -58,48 +62,101 @@ New Standard Library Features
 
 New Package Module Features
 ---------------------------
+* added a new package module supporting calling to Python from Chapel  
+  (see https://chapel-lang.org/docs/2.3/modules/packages/Python.html)
 
 Changes / Feature Improvements in Standard Libraries
 ----------------------------------------------------
 
 Changes / Feature Improvements in Package Modules
 -------------------------------------------------
+* expanded `PrecisionSerializer` to properly handle `complex` & `imag` values  
+  (see https://chapel-lang.org/docs/2.3/modules/packages/Python.html)
```

Comment:
This link should be to PrecisionSerializer

Reply:
Thanks!  (and after I asked you about it and removed the `#` tag… sheesh!)